### ISSUE TITLE
Starting an experiement with a `summary=True` delays execution

### ISSUE THREAD

**Issue by b-fg** [CrayLabs/SmartSim#294]
When calling an experiment to start with `summary=True` enabled, a count down is forced before running the experiment here
https://github.com/CrayLabs/SmartSim/blob/5817084bee1cb96a46e60b9a8821dc9896d3ffe8/smartsim/experiment.py#L826-L827
It would be nice to be able to print the summary without having to wait.

**Comment by ashao** [comment]
Nice spot! I vaguely recall that we had a delay at some point (years ago) to allow time for the orchestrator to spin up, but I think we fixed that problem in a much better way than hardcoding a sleep time and just neglected to remove these lines from the codebase

**Comment by ashao** [comment]
Ok, so checking in with others on the team was that the wait was a "last chance" cancel the launch of the experiment. If you want to see the summary before you start you can always call the `summary` method on your `Experiment` object. Does that suffice in the interim?

**Comment by b-fg** [comment]
Okay! That `summary` produces a slightly different output though. So maybe that can be updated with more info as when using `summary=True`:

`Experiment.summary`:
```
'|    | Name                | Entity-Type   | JobID   | RunID   | Time   | Status    | Returncode   |\n|----|---------------------|---------------|---------|---------|--------|-----------|--------------|\n| 0  | Sod2D_RL_ensemble_0 | Model         | 860440  | 0       | 5.7321 | Completed | 0            |'
```

`summary=True`:
```
=== Launch Summary ===
Experiment: sod2d_exp
Experiment Path: /home/b-fg/Workspace/bsc1/SOD2D_DRL/relexi_sod/examples/bl/sod2d_exp
Launcher: local
Database Status: active

=== Ensembles ===
Sod2D_RL_ensemble
Members: 1
Batch Launch: False
```

**Comment by b-fg** [comment]
Not blocking at all - no rush. I just pointed it out because I thought it could be improved. Thank you for the rapid response!

**Comment by mellis13** [comment]
Closing this issue as it was addressed with #298
